website_search_blog_bar/models/res_config_settings.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    blog_search_title = fields.Boolean(
```

Comment:
I dont understand Why it is an option on res company. I'd see better as an option of the end user making the research. Don't you think ?

Reply:
It is just a first approach. We wanted it to be as simple as possible. Of course, it can be improved in the future. Appart from that, what do you think of the module in general? Thank you

Reply:
Not a precise point of view. I'm really not a website expert.
I let other oca members make reviews.